src/routes/+layout.svelte

```diff
@@ -166,7 +166,7 @@
 	{/if}
 </svelte:head>
 
-{#if !$settings.ethicsModalAccepted}
+{#if !$settings.ethicsModalAccepted && $page.url.pathname !== "/privacy"}
```

Comment:
we can generalize the solution by creating a whitelist array if there are other pages the disclaimer modal should not appear. For now, this is good enough imo

Reply:
I think [layout groups](https://kit.svelte.dev/docs/advanced-routing#advanced-layouts-group) are meant for use cases like this if we want to handle it in the future, but indeed no need to overbuild for now 😄

Reply:
At the moment, I can't think of any other page that should not show disclaimer. Can you ?

Reply:
I think that's the only one for now 👀